core/src/main/java/org/elasticsearch/index/seqno/GlobalCheckpointSyncAction.java

```diff
@@ -129,7 +129,7 @@ public String toString() {
         private ReplicaRequest() {
         }
 
-        public ReplicaRequest(PrimaryRequest primaryRequest, long checkpoint) {
+        public ReplicaRequest(ReplicationRequest primaryRequest, long checkpoint) {
```

Comment:
I manually reverted the changes to GlobalCheckpointSyncAction and I missed one :)

I undid it in f7b29d78395c2e13c715f6c167d6e8cbffe28e6e